cartridges/LINK_adyen/cartridge/client/default/js/adyenSecureFields.js

```diff
@@ -0,0 +1,49 @@
+    const configuration = {
+        locale: 'en_US', // Defaults to en_US
+        translations: {} // Override translations
+    };
+
+    const checkout = new AdyenCheckout(configuration);
+    const cardNode = document.getElementById('card');
+
+    getConfigurationSecureFields();
+
+    var originKey = "";
+    var loadingContext = "";
+    function setConfigData(data){
+        originKey = data.adyenOriginKey[Object.keys(data.adyenOriginKey)[0]];
+        loadingContext = data.adyenLoadingContext;
+    };
+
+    function getConfigurationSecureFields() {
+        $.ajax({
+            url: 'Adyen-GetConfigSecuredFields',
+            type: 'get',
+            data: {protocol : window.location.protocol},
+            success: function (data) {
+                setConfigData(data);
+            },
+            complete: function(){
```

Comment:
Without the config data the card component won't even load I think so maybe it would be better to put the functionality from under the complete event handling into the success